setup.sh

```diff
@@ -614,11 +614,10 @@ if [ "$noperlpath" = "" ]; then
 	echo ""
 fi
 
-# Re-generating main
-rm -f $config_dir/.stop-init $config_dir/.start-init $config_dir/.restart-init $config_dir/.restart-by-force-kill-init $config_dir/.reload-init
+# Re-generating main scripts
 echo "Creating start and stop init scripts.."
 # Start main
```

Comment:
N: Nothing to explain for `setup.sh` as the logic is nearly identical to `setup.pl`.